best-practices/documentation/README.md

```diff
@@ -0,0 +1,8 @@
+# Documentation
+
+## Ruby
+Ruby projects should use [Yard](http://yardoc.org/) for documentation purposes.
+
+One benefit is YARD runs a live documentation server for the community, hosting docs for all RubyGems and Github projects. The project is at [RubyDoc.info](http://www.rubydoc.info/) (formerly rdoc.info)
+
+A [useful Yardoc cheatsheet](https://gist.github.com/chetan/1827484)
```

Comment:
Do you know something similar to http://guides.rubyonrails.org/api_documentation_guidelines.html, but for YARD? I like it's focus on the content of the comment, and maybe it just needs to be adapted for YARD.